snap/snapcraft.yaml

```diff
@@ -34,6 +34,18 @@ parts:
     stage:
       - snapcraft-completion
 
+  patchelf:
+    plugin: autotools
+    source: https://github.com/NixOS/patchelf
```

Comment:
we can now use snapcore and it might be wise to do so now.

Reply:
Done.